arrangor/src/main/kotlin/no/nav/amt/tiltak/ansatt/ArrangorerForAnsattQuery.kt

```diff
@@ -27,11 +27,11 @@ class ArrangorerForAnsattQuery(
 		       arrangor.organisasjonsnummer                  AS arrangor_organisasjonsnummer,
 		       arrangor.overordnet_enhet_navn                AS arrangor_overordnet_enhet_navn,
 		       arrangor.overordnet_enhet_organisasjonsnummer AS arrangor_overordnet_enhet_organisasjonsnummer,
-		       rolle.rolle                                     AS rolle
+		       rolle.rolle                                   AS rolle
 		FROM arrangor_ansatt ansatt
 		         JOIN arrangor_ansatt_rolle rolle ON ansatt.id = rolle.ansatt_id
 		         JOIN arrangor arrangor ON rolle.arrangor_id = arrangor.id
-		where ansatt.personlig_ident = :personligIdent
+		WHERE ansatt.personlig_ident = :personligIdent AND rolle.gyldig_til > current_timestamp
```

Comment:
Jeg vil si at denne buggen belyser et problem om at tilgangskontrollen i SQL burde helst gjøres 1 sted i arrangor ansatt repositoriet hvis mulig. Jeg foreslår og hente ut rollene med servicen som vi allerede har laget for å håndtere roller og heller hente ut en liste med arrangører basert på IDene fra rollene og merge dette sammen til DTOen som vi sender til frontend. Det vil si at vi egentlig kan slette denne querien.